modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceEntryLocalServiceImpl.java

```diff
@@ -695,6 +703,8 @@ public CommercePriceEntry updateCommercePriceEntry(
 		commercePriceEntry = _startWorkflowInstance(
 			user.getUserId(), commercePriceEntry, serviceContext);
 
+		_reindexCPDefinition(cpInstance.getCPDefinitionId());
```

Comment:
Same here, we missed the null check. Can you check if cpInstance is null everywhere in the local service?

Reply:
I checked and it doesn't seem to have a null check anywhere, so i assume it's never null. I'll put the null checks just in case